Unable to build pd extension

Hi! When building the pd extension from the 0.5.0 sources for Arch Linux I have to apply a few fixes, as the build fails otherwise:

```
/build/marsyas/src/marsyas-0.5.0/src/marsyas_pd/ibt_pd.cpp: In function ‘void* ibt_pd_new(t_symbol*, int, t_atom*)’:
/build/marsyas/src/marsyas-0.5.0/src/marsyas_pd/ibt_pd.cpp:200:48: error: invalid conversion from ‘const char*’ to ‘char*’ [-fpermissive]
  200 |       x->outPathName = atom_getsymbol(argv+1)->s_name;
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~
      |                                                |
      |                                                const char*
```

If this is cast, there's still a problem with the cmake setup:

```
[ 91%] Linking CXX shared library ../../lib/ibt_pd.pd_linux
cd /build/marsyas/src/marsyas-0.5.0/build/src/marsyas_pd && /usr/bin/cmake -E cmake_link_script CMakeFiles/ibt_pd.dir/link.txt --verbose=1
/usr/bin/c++ -fPIC -std=c++11 -Wall -Wextra -fPIC -O3 -DNDEBUG -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -shared -Wl,-soname,ibt_pd.pd_linux -o ../../lib/ibt_pd.pd_linux CMakeFiles/ibt_pd.dir/ibt_pd.cpp.o CMakeFiles/ibt_pd.dir/IBT.cpp.o  ../../lib/libmarsyas.so --export-dynamic -shared -lmad -logg -lvorbis -lv
orbisfile -lvorbisenc -ljack -lasound -lpthread -lrt -lmad ../../lib/liboscpack.a -lmp3lame -logg -lvorbis -lvorbisfile -lvorbisenc
c++: error: unrecognized command-line option ‘--export-dynamic’; did you mean ‘-export-dynamic’?
make[2]: *** [src/marsyas_pd/CMakeFiles/ibt_pd.dir/build.make:133: lib/ibt_pd.pd_linux] Error 1
make[2]: Leaving directory '/build/marsyas/src/marsyas-0.5.0/build'
make[1]: *** [CMakeFiles/Makefile2:2653: src/marsyas_pd/CMakeFiles/ibt_pd.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/src/marsyas_pd/CMakeLists.txt
+++ b/src/marsyas_pd/CMakeLists.txt
@@ -38,7 +38,7 @@ if(MARSYAS_MACOSX)
 	SET_TARGET_PROPERTIES(ibt_pd PROPERTIES SUFFIX ".pd_darwin")
 elseif(MARSYAS_LINUX)
 	#define --export-dynamic -shared
-	TARGET_LINK_LIBRARIES(ibt_pd ${MARSYAS_LIBRARY} "--export-dynamic -shared")
+	TARGET_LINK_LIBRARIES(ibt_pd ${MARSYAS_LIBRARY} "-export-dynamic -shared")
 	#define so extension as .pd_linux in Linux 
 	SET_TARGET_PROPERTIES(ibt_pd PROPERTIES SUFFIX ".pd_linux")
 elseif(MARSYAS_WIN32)	

--- a/src/marsyas_pd/ibt_pd.cpp
+++ b/src/marsyas_pd/ibt_pd.cpp
@@ -197,7 +197,7 @@ ibt_pd_new (t_symbol *s, int argc, t_atom *argv)
     else if (!strcmp(firstarg->s_name, "@outPathName") && argc > 1)
     {
       post("ibt_pd %s: %s", firstarg->s_name, atom_getsymbol(argv+1)->s_name);
-      x->outPathName = atom_getsymbol(argv+1)->s_name;
+      x->outPathName = const_cast<char*>(atom_getsymbol(argv+1)->s_name);
       argc -= 2; argv += 2;
     }
     else if (!strcmp(firstarg->s_name, "@stateRecovery") && argc > 1)

